scylla/src/transport/connection.rs

```diff
@@ -1559,8 +1558,7 @@ mod tests {
     /// 1. SELECT from an empty table.
     /// 2. Create table and insert ints 0..100.
     ///    Then use query_all with page_size set to 7 to select all 100 rows.
-    /// 3. INSERT query_all should have None in result rows.
-    /// 4. Calling query_all with a Query that doesn't have page_size set should result in an error.
+    /// 3. INSERT query_all should work and not return any rows.
```

Comment:
I can't see any commit replacing refs to `query_all` in comments with refs to `query_iter`.

Reply:
Done (in the commit which adjusts the test to use query_iter).